Rename parsedData to features

gchoqueux opened the pull request:
## Description
Rename **`parsedData`** to **`features`**.

gchoqueux committed:
fix(examples): replace deprecated parsedData by features.

zarov reviewed (commented):
Looking good, although I am not sure about the `feature` name. Singular for `parsedData` made sense, but less for `feature`. Parsers return a `FeatureCollection` or an array of `Features` most of the time. What do you thing about `features`, plural ?

gchoqueux committed:
chore: rename parsedData by features.

Because parsedData are instances of Feature or FeatureCollection.

FileSource.parsedData is deprecated replace by FileSource.features.

gchoqueux committed:
fix(test): replace deprecated parsedData by features.

gchoqueux commented:
> Looking good, although I am not sure about the `feature` name. Singular for `parsedData` made sense, but less for `feature`. Parsers return a `FeatureCollection` or an array of `Features` most of the time. What do you thing about `features`, plural ?

Yes, it's done
